### ISSUE TITLE
Generate valid HTML

### ISSUE THREAD

**Issue by nikataus** [mistic100/jQuery-QueryBuilder#490]
Hi again,

My issue is related more to provide accessibility for the application. The issue is that when you add new group you create new DL element and append it to UL which violates some standards. I understand that is not the goal here, but I just wanted to check with you to see if there is alternative way to wrap DL within LI element and get it to work. I tried it, but I got some errors. Please let me know if you have any ideas? 
Thanks!

**Comment by mistic100** [comment]
It's indeed ugly... don't know why I did it like this. DL doesn't even make sense as there is only one DT/DD

It should work by replacing the DL by a LI and the DT add DD by two DIVs, and also add a main UL wrapper when creating the builder. I will try

**Comment by nikataus** [comment]
Thanks for your quick reply! Can you give me any date this could be released?
